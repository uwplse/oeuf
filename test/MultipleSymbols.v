Require List SourceLang Pretty OeufPlugin.OeufPlugin CompilationUnit HList.
Import HList List.ListNotations.

Eval lazy Then Write To File "multiplesymbols.oeuf"
     (Pretty.compilation_unit.pretty 75
       (CompilationUnit.CompilationUnit _ (hcons (@SourceLang.add_reflect [])
                                          (hcons (@SourceLang.fib_reflect []) hnil)))).
