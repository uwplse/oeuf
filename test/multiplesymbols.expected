fib(0) = 0
add(0, 0) = 0
fib(1) = 1
add(1, 1) = 2
fib(2) = 1
add(2, 2) = 4
fib(3) = 2
add(3, 3) = 6
fib(4) = 3
add(4, 4) = 8
fib(5) = 5
add(5, 5) = 10
fib(6) = 8
add(6, 6) = 12
fib(7) = 13
add(7, 7) = 14
